src/repart/repart.c

```diff
@@ -8804,20 +8803,21 @@ static int parse_argv(int argc, char *argv[], X509 **ret_certificate, EVP_PKEY *
                         break;
 
                 case ARG_KEY_FILE: {
-                        _cleanup_(erase_and_freep) char *k = NULL;
-                        size_t n = 0;
+                        struct iovec key = { 0 };
```

Comment:
Just `{}`.

Reply:
Unaddressed. Please use `{}`.

Reply:
Still not addressed. Please fix it in the first commit.